prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+   <a href= "index.html"> <div> Click here to go to the home page </div> </a>
```

Comment:
Be sure to include the `".."` as well so that the browser loads the `index.html` file from the parent directory.

```html
<a href="../index.html"> etc etc etc </a>
```